widgetssdk/src/main/java/com/glia/widgets/view/VisitorCodeView.kt

```diff
@@ -100,6 +99,9 @@ class VisitorCodeView internal constructor(
         runOnUi {
             showProgressBar(true)
             showSuccess()
+            successTitle.contentDescription = context.getString(R.string.glia_visitor_code_loading)
+            successTitle.sendAccessibilityEvent(AccessibilityEvent.TYPE_VIEW_ACCESSIBILITY_FOCUSED)
```

Comment:
Can you please add some comments inside the code for this line. Even checking the [property doc](https://developer.android.com/reference/android/view/accessibility/AccessibilityEvent#TYPE_VIEW_ACCESSIBILITY_FOCUSED) I don't understand why it is needed.

Reply:
Added comment